qutebrowser/mainwindow/prompt.py

```diff
@@ -535,6 +542,7 @@ def __init__(self, question, parent=None):
         self._vbox = QVBoxLayout(self)
         self._vbox.setSpacing(15)
         self._key_grid = None
+        self._base_commands = [('prompt-toggle', 'Toggle prompt\nvisibility')]
```

Comment:
Since this is a constant, it should be defined on class level, like `KEY_MODE` (as `_BASE_COMMANDS`).

Also, I think I'd prefer dropping the embedded `\n` here. It seems easier to scan the contents visually if one keybinding is one line:

![image](https://user-images.githubusercontent.com/625793/196029383-0d262629-7690-4511-b14a-105c9ff3c15f.png)

Reply:
Hi I have addressed all the suggested changes. Thank you.